src/display/annotation_storage.js

```diff
@@ -120,7 +120,25 @@ class AnnotationStorage {
     return this.#storage.has(key);
   }
 
+  /**
+   * @param {Object} obj
+   */
+  setAll(obj) {
+    if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")) {
```

Comment:
I'm not sure it's a good idea, because we need to remember that this function is not available in m-c.

Reply:
I've reverted that now, since it's not a lot of code.

(However I don't think we should be using this method in the main code-base, given the "duplicated" iteration.)

Reply:
Why not prefix them with a double underscore to help to remind us (or people in the future) that those methods are kinda private/internal ?

Reply:
There might be third-party use-cases where it's helpful if we provide a slightly simpler way of pre-populating the `AnnotationStorage` before rendering, I'm reminded of e.g. issue #15712, hence I'll leave the names as-is.